modules/metering-store-data-in-s3.adoc

```diff
@@ -119,3 +119,5 @@ If you left `spec.storage.hive.s3.createBucket` set to `true`, or unset, then yo
     ]
 }
 ----
+
+Metering does not manage or delete any s3 bucket data. When uninstalling metering, any s3 buckets created for metering storage must be manually cleaned up.
```

Comment:
I think we should move this higher up to avoid the situation where a user skips over some key information.

Reply:
I think we should also specify that this applies to any s3 bucket used for metering and not just buckets created by Metering, i.e. "..., any s3 buckets used to store metering data must be manually cleaned up."

Reply:
Sounds good. I made some updates per your feedback.